src/processing/IChanges.java

```diff
@@ -12,7 +12,6 @@
  */
 
 public interface IChanges {
```

Comment:
@sakshi1603 @himanshu2047 @convict-git @muhammedyaseen-git 
Minor Comment: What is the coding convention for interfaces? Should they begin with "I" or not? Here it is "IChanges" (with an I), but I found another interface called "BoardObjectOperation" (without an I). Not a big deal but let's keep it consistent across the board (such as for ICommunicator). And if we are too far into the coding, dont worry about it - just calling it out.

Is there a way to automatically enforce our coding conventions and static code analysis (like FxCop/StyleCop for C#)?

Reply:
Everywhere else it's with 'I', will ask Ahmed to use the same.

Reply:
I think @tymefighter already changed it to `IBoardObjectOperation` everywhere.

Reply:
Yes Sir @tymefighter has changed name of board_object as well as included prefix I for interface. Like @himanshu2047 said somehow you watched older files.